github/event_types.go

```diff
@@ -752,6 +793,19 @@ type RepositoryVulnerabilityAlertEvent struct {
 	} `json:"alert,omitempty"`
 }
 
+// StarEvent is triggered when a star is added or removed from a repository.
+// The Webhook event name is "star".
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#starevent
+type StarEvent struct {
+	// Action is the action that was performed. This can be: "created" or "deleted".
+	Action *string `json:"action,omitempty"`
+
+	// The time the star was created. This is a timestamp in ISO 8601 format: YYYY-MM-DDTHH:MM:SSZ.
```

Comment:
Since our internal timestamp parsing code handles the format for us, I think we should remove the "This is a timestamp... SSZ." sentence.

Reply:
Also, for the godocs, let's say `// StarredAt is the time the star was created. It will be null for the "deleted" action.`